src/utils/StorageUtils.ts

```
@@ -0,0 +1,245 @@
+  url: string;
+  path: string;
+  fileName: string;
+  metadata?: Record<string, any>;
+}
+
+/**
+ * Validate if the provided object is a valid File or Blob
+ * @param file - The file to validate
+ * @returns boolean indicating if file is valid
+ */
+const isValidFile = (file: any): boolean => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)